relayer/pkg/chainlink/txm/txm.go

```diff
@@ -193,9 +200,9 @@ func (txm *starktxm) Ready() error {
 	return txm.starter.Ready()
 }
 
-func (txm *starktxm) Enqueue(tx types.Transaction) error {
+func (txm *starktxm) Enqueue(sender caigotypes.Hash, tx caigotypes.FunctionCall) error {
 	select {
-	case txm.queue <- tx:
+	case txm.queue <- Tx{sender: sender, call: tx}:
```

Comment:
just a thought: would it be easier to just pass the `Tx` struct rather than passing two parameters?

i'm thinking in a future case where we might add more parameters, then we can just define them in the struct rather than changing the function signature